docs/guides/parameters.md

```diff
@@ -13,59 +13,116 @@ layout: Doc
-sls param list
+
+### Creating Serverless Dashboard parameters
+
+Parameters can be created in the [Dashboard](https://app.serverless.com/) at the service level (applies to all stages) or instance level (stage-specific).
+
+To manage parameters on a service, go to the **apps** section of the dashboard, and select **settings** under the **...** menu.
+
+To manage parameters on an instance, go to the **app** section of the dashboard, select the instance, and go to the **params** tab.
+
+### Retrieving parameters from the command line
+
+Dashboard parameters can also be accessed on the CLI. You can use this at development time to look up the parameters without opening the dashboard, or in your CI/CD pipeline to use the parameters in custom scripts.
+
+#### List parameters
+
+If you are in a directory with a `serverless.yml`, the parameters will be listed for the org, app, and service specified in the `serverless.yml` file:
+
+```bash
+serverless param list [--stage <stage>]
+```
+
+If you are in a directory without a `serverless.yml`, or if you want to access parameters from another org, app, service, stage, or region, you can pass in the optional flags:
+
+```bash
+serverless param list
   [--org <org>]
   [--app <app>]
   [--service <service>]
   [--stage <stage>]
   [--region <region>]
 ```
 
-If you are in a working directory with a `serverless.yml` then the parameters will be listed for the org, app, and service specified in the `serverless.yml` file.
+#### Get a parameter
 
-If you are not in a working directory, without a `serverless.yml`, or if you want to access parameters from another org, app, service, stage, or region, you can pass in the optional flags.
-
-### Get a parameter
+Individual parameters can also be accessed from the CLI using the `param get` sub-command. This command requires the `--name <name>` flag to identify the parameter name. Like the `sls param list`, you can optionally specify a different org, app, service, stage, ore region using flags.
 
-```
-sls param get
-  --name <name>
+```bash
+serverless param get --name <name>
   [--org <org>]
   [--app <app>]
   [--service <service>]
   [--stage <stage>]
   [--region <region>]
 ```
 
-Individual parameters can also be accessed from the CLI using the `param get` sub-command. This command requires the `--name <name>` flag to identify the parameter name. Like the `sls param list`, you can optionally specify a different org, app, service, stage, ore region using flags.
+## Inheritance and overriding
+
+Parameters can be defined in `serverless.yml` per stage, as well as in Serverless Dashboard on the service or the instance (stage). Here is the priority used to resolve a parameter:
+
+- First, look in `params.<stage>` in `serverless.yml`
+- If not found, then look in the instance's parameters in the Dashboard
+- If not found, then look in `params.default` in `serverless.yml`
+- If not found, then look in the service's parameters in the Dashboard
+- If not found, throw an error
```

Comment:
When variables resolution is concerned it's not precisely true. On missing values error is thrown only if no fallback value is provided.

So more accurate would be to simply skip that line, or explain more precisely what happens depending on the context

Reply:
@medikoo do you mean:

- `${param:foo}` will throw an error
- `${param:foo, 'default'}` will resolve to `default`

?

Reply:
Yes, exactly

Reply:
👍  I clarified that, thanks